### PR TITLE
Signup: Testing Flow for Improved Onboarding

### PR DESCRIPTION
**Pull request by michaeldcain**
**Do Not Merge**

This is a testing flow for the Improved Onboarding usability tests. It is a clickable prototype and does not result in a site being created.

screen shot 2018-10-25 at 11 41 23 am

### CODE REVIEW

File: client/signup/steps/business-information/index.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+					<form onSubmit={ this.handleSubmit }>
+						<Card>
+							<h3>{ translate( 'Basic Information' ) }</h3>
+							{ translate( 'How do people find and contact your business?' ) }
+
+							<FormFieldset>
+								<FormTextInput
+									id="name"
+									name="name"
+									placeholder={ translate( 'Business Name' ) }
```
Comment (a8ci18n) [inline client/signup/steps/business-information/index.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Business name' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/signup/steps/business-information/index.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+			<div className="business-information__wrapper">
+				<div className="business-information__form-wrapper ">
+					<form onSubmit={ this.handleSubmit }>
+						<Card>
+							<h3>{ translate( 'Basic Information' ) }</h3>
+							{ translate( 'How do people find and contact your business?' ) }
+
+							<FormFieldset>
+								<FormLabel htmlFor="name">
+									{ translate( 'Business Name' ) }
```
Comment (a8ci18n) [inline client/signup/steps/business-information/index.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Business name' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/signup/steps/business-needs/index.jsx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+		// Note that the key attributes will be used in the name of a tracks event attribute so can not
+		// contain whitespace.
+		const options = {
+			contact: {
+				key: 'contact',
+				formLabel: translate( 'Allow people to contact me' ),
+			},
+			payments: {
+				key: 'payments',
+				formLabel: translate( 'Collect payments' ),
```
Comment (a8ci18n) [inline client/signup/steps/business-needs/index.jsx]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Collect Payments and Donations' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/signup/steps/business-needs/index.jsx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+				key: 'share',
+				formLabel: translate( 'Share your work' ),
+			},
+			capture: {
+				key: 'capture',
+				formLabel: translate( 'Collect emails' ),
+			},
+		};
+
+		const businessNeedsArray = [
```
Comment (kwonye) [inline client/signup/steps/business-needs/index.jsx]: Is there a plan to send these through the API? I believe we need it to be so on the mobile apps to make sure we consistently send the correct one regardless of which version we're at.

I'm thinking in a few years when we add functionality we don't have now, we want that to be available to the apps.

Comment (michaeldcain) [reply client/signup/steps/business-needs/index.jsx]: We're thinking that these will eventually be tailored to the vertical selection, so yes, we'll try to return it from an endpoint.

**Comment by michaeldcain** [comment]
These tests were run and wrapped up. Closing.

**Pull request closed without merging**